dev/bots/suite_runners/run_android_engine_tests.dart

```diff
@@ -86,23 +86,49 @@ Future<void> runAndroidEngineTests({required ImpellerBackend impellerBackend}) a
       // final Match? stdoutMatch = impellerStdoutPattern.firstMatch(stdout);
       // if (stdoutMatch == null) {
       //   foundError(<String>['Could not find pattern ${impellerStdoutPattern.pattern}.', stdout]);
-      //   continue;
+      //   return;
       // }
 
       // final String reportedBackend = stdoutMatch.group(1)!.toLowerCase();
       // if (reportedBackend != impellerBackend.name) {
       //   foundError(<String>[
       //     'Reported Imepller backend was $reportedBackend, expected ${impellerBackend.name}',
       //   ]);
-      //   continue;
+      //   return;
       // }
     }
+
+    for (final FileSystemEntity file in mains) {
+      if (file.path.contains('hcpp')) {
+        continue;
+      }
+      await runTest(file);
+    }
+
+    // Test HCPP Platform Views
+    androidManifestXml.writeAsStringSync(
```

Comment:
You'd want to skip this stanza for OpenGLES.

Reply:
Done